capstone/src/main/webapp/form-loader.js

```diff
@@ -48,21 +48,21 @@ function getForms() {
         const formElement = document.getElementById("form-container");
         formElement.innerHTML = "";
         formsList.forEach((form) => {
-            formElement.appendChild(createTeacherFormElement(form.editURL));
-            formElement.appendChild(createStudentFormElement(form.URL)); 
+            formElement.appendChild(createTeacherFormElement(form.entity.propertyMap.editUrl));
+            formElement.appendChild(createStudentFormElement(form.entity.propertyMap.Url)); 
         });
     });
 }
 
-function createTeacherFormElement(editURL) { 
+function createTeacherFormElement(editUrl) { 
     const aElement = document.createElement("a");
     aElement.innerText = "Form Edit Page";
-    aElement.href = editURL;
+    aElement.href = editUrl;
     return aElement;
 }
 
-function createStudentFormElement(URL) {
+function createStudentFormElement(Url) {
```

Comment:
for variable naming, you still start with lowercase even if it is an acronym. (you got the editUrl one right though)
(applies everywhere)